src/recipe/recipe.service.ts

```diff
@@ -8,16 +8,33 @@ import { FindRecipeByIdQuery } from './queries/impl/findRecipeById.query';
 import { CreateRecipeCommand } from '../recipe/commands/impl/createRecipe.command';
 import { UpdateRecipeCommand } from '../recipe/commands/impl/updateRecipe.command';
 import { FindAllRecipesQuery } from '../recipe/queries/impl/findAllRecipes.query';
+import { NotificationsGateway } from '../gateways/notification.gateway';
 
 @Injectable()
 export class RecipeService {
-  constructor(private queryBus: QueryBus, private commandBus: CommandBus) {}
+  constructor(
+    private queryBus: QueryBus,
+    private commandBus: CommandBus,
+    private notificationsGateway: NotificationsGateway,
+  ) {}
 
   async createRecipe(
     recipe: CreateRecipeDTO,
     authorId: number,
   ): Promise<RecipeResponse> {
-    return this.commandBus.execute(new CreateRecipeCommand(recipe, authorId));
+    let notificationPayload;
+
+    const recipeFromDb = await this.commandBus.execute(
+      new CreateRecipeCommand(recipe, authorId),
+    );
+
+    if (recipeFromDb) {
+      notificationPayload = 'Recipe created succesfully';
+    }
+
+    this.notificationsGateway.server.emit('notification', notificationPayload);
```

Comment:
Note that you are using server instance directly from notifications Gataway and then you emit the message.
Usually, it would be a good idea to create some kind of module that will handle notifications.
Imagine that there are more kind of notifications. email, push etc and the notifications should have some specific content.

So:
please add some kind of nestjs module that will have method like: sendRecipeCreatedNotifications()

Only this module should use the websockets gateway to handle sending notifications.
this module's methods should be available for other services.
When recipe is created ReicipeService should use this new module to send notifications